dino.icu.yaml

```diff
@@ -66,7 +66,12 @@ art: # by https://github.com/rajanwastaken
   ttl: 1
   type: CNAME
   value: particle-generativeart.netlify.app.
-
+  
+canteen:
+  ttl: 1
+  type: A
+  value: 76.76.21.21 # vercel
+  
```

Comment:
If this is vercel, this should be a CNAME to `cname.vercel-dns.com` with a ttl of 600.

Reply:
I'm unsure if this will work as there is only an option for A record but I'll give it a try

Reply:
<img width="1008" alt="image" src="https://github.com/hackclub/dns/assets/81070048/72b87055-91e2-4ca1-b523-7f5ae601e980">

i don't see any verification step